stores/albert.js

```diff
@@ -0,0 +1,142 @@
+    ); // sha256Hash = '8b68b8590c7d24f3ed8e338aa42e94e7d741766744bb9b9c87e15e18f332e4e5';
+
+    for (let levels of categories.data.leftHandNavigationBar.levelInfo) {
+        let levelStarts = 1;
+        let page = 0,
+            remains = 0;
+        do {
+            const query = `GetCategoryProductSearch&variables={"lang":"cs","searchQuery":"","category":"${levels.code}","pageNumber":${page},"pageSize":${blockOfPages},"filterFlag":true,"plainChildCategories":true}&extensions={"persistedQuery":{"version":1,"sha256Hash":"${sha256Hash}"}}`;
+            const res = await axios.get(ALBERT_BASE_URL + query, {
+                validateStatus: function (status) {
+                    return (status >= 200 && status < 300) || status == 429;
+                },
+            });
+            if (levelStarts) {
+                remains = res.data.data.categoryProductSearch.pagination.totalResults;
+                levelStarts = 0;
+            }
+
+            // exponential backoff
+            backoff = 2000;
+            while (res.status != 200) {
+                console.info(`Albert API returned ${res.status}, retrying in ${backoff / 1000}s.`);
+                await new Promise((resolve) => setTimeout(resolve, backoff));
+                backoff *= 2;
```

Comment:
Tady bych to nejak ohranicil aby to nebylo vetsi nez treba 86400 (jeden den). Ale OK.